### ISSUE TITLE
Form color theme options are not accessible via keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1815]
Inside Forms, the color theme options are not accessible when using the keyboard:

![Image](https://github.com/user-attachments/assets/6c0d69f8-3079-4bbe-83c2-81863fe242db)
